platform/darwin/src/MGLAccountManager.m

```diff
@@ -91,7 +91,7 @@ + (void)setAccessToken:(NSString *)accessToken {
 
 #if TARGET_OS_IPHONE || TARGET_OS_SIMULATOR
     dispatch_async(dispatch_get_main_queue(), ^{
-        [MGLMapboxEvents setupWithAccessToken:accessToken];
+        [MGLMapboxEvents setAccessToken:accessToken];
```

Comment:
This is the only place in the SDK where `setupWithAccessToken` - surely calling this is still required? Or are you saying it needs to be elsewhere?

Reply:
Yeah, I think we need to find a better place for `setupWithAccessToken`. It should really only be called once.